mqtt3/1.0.0/modules/ROOT/pages/mqtt3-connector-listener.adoc

```diff
@@ -0,0 +1,121 @@
+= Listen for New Messages with MQTT3 Connector
+
+The Anypoint Connector for MQTT3 (MQTT3 Connector) *On New Message* (`<mqtt3:listener>`) source enables you to listen for new incoming messages using one or more topic filters (`topicFilter`). Each topic filter has a specific quality of service (`qos`) configured.
+
+In the Configuration XML editor, the `<mqtt3:listener>` and `<mqtt3:topic>` configurations look like this:
+
+[source,xml,linenums]
+----
+    <flow name="listenerAuthorQuotes">
+        <mqtt3:listener config-ref="MQTT_Config">
+            <mqtt3:topics>
+                <mqtt3:topic topicFilter="quotes/terryPratchett" qos="EXACTLY_ONCE"/>
+                <mqtt3:topic topicFilter="quotes/neilGaiman" qos="AT_LEAST_ONCE" />
+                <mqtt3:topic topicFilter="quotes/ianMcEwan" qos="AT_MOST_ONCE" />
+            </mqtt3:topics>
+        </mqtt3:listener>
+    </flow>
+----
+
+== Configure Single-Level Wildcards in Topic Filters
+
+Configure single-level wildcards to listen to multiple topics. Single-level wildcards (`+`) enable the subscriber to receive messages published to all topics matching a specific structure.
+
+In the following example, you configure the `topicFilter` parameter with the structure `quotes/+/authors`:
+
+[source,xml,linenums]
+----
+    <flow name="quotes">
+        <mqtt3:listener config-ref="${config}">
+            <mqtt3:topics>
+                <mqtt3:topic topicFilter="quotes/+/authors"/>
+            </mqtt3:topics>
+        </mqtt3:listener>
+
+        <logger level="DEBUG" message="A quote has been published to #[attributes.topic]: #[payload]"/>
+    </flow>
+----
+
+Messages published to the following topics trigger the listener:
+
+* `quotes/british/authors`
+* `quotes/american/authors`
+
+However, messages published to the following topics do not trigger the listener:
```

Comment:
I really like that you give examples for what triggers the listener and what doesn't trigger the listener.